lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -207,43 +207,50 @@ redistribute_policy:
     config_set_append: ' <state> redistribute <protocol> route-policy <policy>'
 
 #config in vrf context
+# TODO: fix these for XR
 route_target_both_auto:
```

Comment:
If this property is broken for XR, you should add `_exclude: [cli_ios_xr]`

Also, why is `config_get` listed underneath `cli_nexus`?  If `cli_ios_xr` is excluded, you don't need the `cli_nexus` specification.